apis/Google.Devtools.AspNet/Google.Devtools.AspNet.Tests/CloudTrace/TraceHeaderContextTest.cs

```diff
@@ -0,0 +1,112 @@
+﻿// Copyright 2016 Google Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+using Moq;
+using System.Collections.Specialized;
+using System.Web;
+using Xunit;
+
+namespace Google.Devtools.AspNet.Tests
+{
+    public class TraceHeaderContextTest
+    {
+        private static readonly HttpRequest Request = new HttpRequest("file_name", "https://www.google.com", "");
+        private static readonly string TraceId = "105445aa7843bc8bf206b12000100f00";
+        private static readonly ulong SpanId = 81237123;
+
+        /// <summary>
+        /// Create the string trace header value with an optional mask.
+        /// </summary>
+        private string CreateTraceHeaderValue(int? mask = null)
+        {
+            string headerValue = TraceId + "/" + SpanId + ";";
```

Comment:
It's pretty much the same, but `string headerValue = $"{TraceId}/{SpanId};";` is a bit cleaner IMO. Up to you.